api/catalog/api/serializers/media_serializers.py

```diff
@@ -345,6 +386,11 @@ class Meta:
         source="identifier",
     )
 
+    created_on = serializers.DateTimeField(
+        format="%Y-%m-%d",
```

Comment:
When testing I actually had it set to the same format (with time information) but I'm not 100% certain if that is information we want to surface in the API. So I stuck to @zackkrida's original implementation which used `DateField`. Tagging him for clarification on why that field was used and if there are any concerns against showing the precise time.

Reply:
> clarification on why that field was used 

@dhruvkb I really don't recall, it's been some time since I looked at this. Including time information seems like an improvement to me.

Reply:
Awesome, increased precision in 650d70b.